scripts/build

```diff
@@ -33,87 +31,87 @@ REQ_PACKAGES="build-pkgs-$COMPONENT.list"
 [ -r "$ORIG_SRC/build-deps-$MAKE_TARGET_ONLY.list" ] && REQ_PACKAGES="$ORIG_SRC/build-deps-$MAKE_TARGET_ONLY.list"
 
 export USER_UID=$UID
-if ! [ -d chroot-$DIST ]; then
+if ! [ -d "chroot-$DIST" ]; then
     echo "ERROR: You need to have build environment prepared first,"
     echo "compile any other component (for the same target distribution) first"
 fi
 
-if [ -r $REQ_PACKAGES ] && [ $REQ_PACKAGES -nt chroot-$DIST/home/user/.installed_${COMPONENT}_`basename $REQ_PACKAGES` ]; then
-    sed "s/DIST/$DIST/g" $REQ_PACKAGES > build-pkgs-temp.list
+if [ -r "$REQ_PACKAGES" ] && [ "$REQ_PACKAGES" -nt "chroot-$DIST/home/user/.installed_${COMPONENT}_$(basename "$REQ_PACKAGES")" ]; then
+    sed "s/DIST/$DIST/g" "$REQ_PACKAGES" > build-pkgs-temp.list
     echo "-> Installing $COMPONENT build dependencies in $DIST environment"
-    sudo chroot $PWD/chroot-$DIST yum install -y `cat build-pkgs-temp.list`
+    sudo chroot "$PWD/chroot-$DIST" yum install -y "$(cat build-pkgs-temp.list)"
```

Comment:
This is wrong, the package list should be given as separate arguments (so - no ").

Reply:
Good catch

Reply:
Fixed